modules/das-operator-viewing-allocationclaims.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
```

Comment:
The content of this module can be merged with the module for the troubleshooting.

Reply:
I'm going to merge this into the section "Monitoring AllocationClaims", because some of the commands are the same.